### PR TITLE
Clarify that we do not offer atmospheric correction options

### PR DESCRIPTION
**Pull request by hjkristenson**
In the error section, we mention that it is possible to apply an atmospheric correction to the data. This change clarifies that we do not offer any of those corrections as part of the On Demand processing. 

It would be great if we could link to other software that can be used to perform these corrections on the GAMMA-generated products. Is it appropriate to add in links to GACOS http://www.gacos.net/ or TRAIN http://davidbekaert.com/download/TRAIN_manual.pdf 

I would LOVE to have a workflow or guidance that walks though using the HyP3 InSAR products in an atmospheric correction workflow.

### CODE REVIEW

**Review by hjkristenson — COMMENTED** [review]
This commit was supposed to have the message: 
add text clarifying that we do not offer error correction workflows

File: docs/guides/insar_product_guide.md [hunk 1]
```diff
@@ -249,6 +236,7 @@ The local incidence angle is defined as the angle between the incident radar sig
 The baseline is defined as the difference of the platform positions when a given area is imaged. HyP3 baselines are calculated using the best state vectors available. These are either restituted or precise. **If no restituted or precise state vectors are available, the process will not run**.
 
 ## Error Sources
+There are a number of potential sources for error in InSAR products, and these may be present in the On Demand products. We do not currently offer the option to apply corrections to On Demand InSAR products, but further processing or time series analysis can be performed by the user to identify or reduce the impact of errors when using InSAR for analysis.
```
Comment (hjkristenson) [inline docs/guides/insar_product_guide.md:239]: This commit was supposed to have the message: 
add text clarifying that we do not offer error correction workflows

File: docs/guides/insar_product_guide.md [hunk 2]
```diff
@@ -40,25 +40,12 @@ The term *baseline* refers to the physical distance between the two vantage poin
 
 To monitor surface deformation, the perpendicular baseline for the two acquisitions should be very small in order to maximize the coherence of the phase measurements. 
 
-In order to determine topography, two slightly different vantage points are required. Sensitivity to topography depends on the perpendicular baseline, the sensor wavelength, the distance between the satellite and the ground, and the sensor look angle as given in equation 1. The parameters are diagrammed in Figure 3. 
+In order to determine topography, two slightly different vantage points are required. Sensitivity to topography depends on the perpendicular baseline, the sensor wavelength, the distance between the satellite and the ground, and the sensor look angle.
```
Comment (hjkristenson) [inline docs/guides/insar_product_guide.md:43]: This commit was not supposed to be included in this PR; I'd pushed it to a different branch (topo-remove) to deal with it separately, but I'm not sure if it's worth splitting it out at this point.

File: docs/guides/insar_product_guide.md (hunk 1 above)
Comment (asjohnston-asf) [reply docs/guides/insar_product_guide.md:239]: Maybe this?

On Demand InSAR products do not currently correct for some common sources of error in interferometry, but further processing or time series analysis can be performed by the user to identify or reduce the impact of errors when using InSAR for analysis.

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**